traffic_control/tests/util.py

```diff
@@ -0,0 +1 @@
+REQUIRED_HEADERS = {"HTTP_USER_AGENT": "test-client"}
```

Comment:
Como essa alteração relacionada ao User-Agent afeta virtualmente todas as requisições, acredito que uma abordagem melhor seria [personalizar o test client](https://docs.djangoproject.com/en/3.1/topics/testing/tools/#customizing-the-test-client) (seja criando uma nova classe ou uma instância da classe padrão que já tenha esse cabeçalho).

Reply:
Resolvido em 9679932